app/views/stats/index.html.erb

```diff
@@ -17,9 +17,13 @@
 
   <p><%= raw t('notes.stats.looking_back', :time => @time.to_formatted_s(:long)) %></p>
   <p>
-    <%= raw t('notes.stats.try_time_further_back', :url1 => "?time="+(@time-1.year).to_formatted_s(:long), 
+    <%= raw t('notes.stats.try_time_back', :url1 => "?time="+(@time-1.year).to_formatted_s(:long),
```

Comment:
@kevinzluo, there is no need to correct this variable here. If there is some issue with this variable, please change it in another PR as we follow one type of change in one PR.